index.html

```diff
@@ -31,6 +31,14 @@
             </div>
         </nav>
     </header>
+    <nav class="mobile-nav ">
+        <span class="close-button">x</span><br><br>
+        <ul class="menu-list-mobile" >
+            <li><a class="menu-list-item" href="#works">Portfolio</a></li><br>
+            <li><a class="menu-list-item" href="#about-myself">About</a></li><br>
+            <li><a class="menu-list-item" href="#my-form">Contact</a></li><br>
+        </ul>
+    </nav>
```

Comment:
- Kindly make sure to stick as much as possible to the Figma template, which includes:
    - [ ] Changing the background color of the "close" button and adjusting its size.
    - [ ] Applying a background filter to match the original design,

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/223117509-9cd77295-a848-411f-a3e5-9f6eee8e4696.png) | ![image](https://user-images.githubusercontent.com/62290545/223118075-4ef8c61e-3a36-4783-86f0-72e84c4e8da8.png) |

Reply:
done